lib/json/serialization.nit

```diff
@@ -448,6 +448,25 @@ class JsonDeserializer
 	end
 end
 
+redef class Text
+
+	# Deserialize a `nullable Object` from this JSON formatted string
+	#
+	# Warning: Any deserialization errors are lost and may return `null` or a partial object.
+	# For this reason, this method should only be used in simple scripts.
+	#
+	# Require: `self` formatted in valid JSON, with deserialization metadata.
+	fun from_json_string: nullable Object
+	do
+		var deserializer = new JsonDeserializer(self)
+		var res = deserializer.deserialize
+		if deserializer.errors.not_empty then
+			print_error "Deserialization errors: {deserializer.errors.join(", ")}"
```

Comment:
I do not really like when libraries print things but I really hate them when they do it without even documenting it.